inference-engine/src/vpu/common/include/vpu/ngraph/operations/static_shape_loop.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/opsets/opset5.hpp>
+
+namespace ngraph { namespace vpu { namespace op {
+
+class StaticShapeLoop : public ngraph::opset5::Loop {
+public:
+    static constexpr NodeTypeInfo type_info{"StaticShapeLoop", 5};
```

Comment:
For all new operations in the nGraph we use `NGRAPH_RTTI_DECLARATION` macro ([example](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/include/ngraph/op/hswish.hpp#L36))

Reply:
Fixed

Reply:
I don't see the use of the macro yet, is this really fixed?

Reply:
Excellent catch! I must have pushed old version before started to fix your comments

Reply:
Fixed